src/ripple/app/main/LoadManager.cpp

```diff
@@ -32,40 +33,56 @@
 
 namespace ripple {
 
-class LoadManagerImp
-    : public LoadManager
-    , public beast::Thread
+class LoadManagerImp : public LoadManager
 {
 public:
-    using LockType = std::mutex;
-    using ScopedLockType = std::lock_guard <LockType>;
+    //--------------------------------------------------------------------------
 
-    Application& app_;
-    beast::Journal m_journal;
-    LockType mLock;
+    Application& mApp;
+    beast::Journal mJournal;
```

Comment:
"A trailing underscore is clearly better than an `m` prefix, and therefore more correct."  You are talking about a style issue only.  There is no "correctness" in your discussion.  I am fine with using a trailing underscore in member names.  The [FOLD] will use trailing underscores.  But, as far as I'm concerned, we're not talking about a code correctness issue.

Why in the world would you _not_ rename existing members to be consistent?  See https://books.google.com/books?id=mmjVIC6WolgC&pg=PT39&lpg=PT39&dq=C%2B%2B+keep+naming+consistent+within+a+file&source=bl&ots=cdSpGLlIT4&sig=ieYtFni7Xap6lzIBgoByDTfAv-4&hl=en&sa=X&ved=0CCkQ6AEwAmoVChMIlaH4hMKzyAIVR5qICh33BgiL#v=onepage&q=C%2B%2B%20keep%20naming%20consistent%20within%20a%20file&f=false.

My inclination is to go with Sutter and Alexandrescu on this issue.  But you're the boss.  If you want to retain the old mixed member names I can do that.  But you won't convince me that it's a "correctness" issue.